src/main/java/co/fairtide/prometheus_aeron_stat/PrometheusAeronStat.java

```diff
@@ -0,0 +1,263 @@
+    private Gauge numSubscriberGauge;
+    private Gauge numClientGauge;
+
+
+    public PrometheusAeronStat(CountersReader counters) {
+        this.counters = counters;
+        this.prometheusCounterMap = new HashMap<>();
+        this.numChannelPublishingGauge = Gauge.build().name("num_channel").help("num_channel").register();
+        this.numSubscriberGauge = Gauge.build().name("num_subscribers").help("num_channel").register();
+        this.numClientGauge = Gauge.build().name("num_clients").help("num_channel").register();
+    }
+
+    public static CountersReader mapCounters() {
+        final File cncFile = CommonContext.newDefaultCncFile();
+        System.out.println("Command `n Control file " + cncFile);
+
+        final MappedByteBuffer cncByteBuffer = IoUtil.mapExistingFile(cncFile, "cnc");
+        final DirectBuffer cncMetaData = createMetaDataBuffer(cncByteBuffer);
+        final int cncVersion = cncMetaData.getInt(cncVersionOffset(0));
+
+        if (CNC_VERSION != cncVersion) {
+            throw new IllegalStateException(
+                    "Aeron CnC version does not match: version=" + cncVersion + " required=" + CNC_VERSION);
+        }
+
+        return new CountersReader(
+                createCountersMetaDataBuffer(cncByteBuffer, cncMetaData),
+                createCountersValuesBuffer(cncByteBuffer, cncMetaData),
+                StandardCharsets.US_ASCII);
+    }
+
+    public static void main(final String[] args) throws Exception {
+        long delayMs = 1000L;
+        int portNumber = -1;
+
+        if (0 != args.length) {
+            checkForHelp(args);
+            for (final String arg : args) {
+                final int equalsIndex = arg.indexOf('=');
+                if (-1 == equalsIndex) {
+                    System.out.println("Arguments must be in name=pattern format: Invalid '" + arg + "'");
+                    return;
+                }
+                final String argName = arg.substring(0, equalsIndex);
+                final String argValue = arg.substring(equalsIndex + 1);
+
+                switch (argName) {
+                    case DELAY:
+                        delayMs = Long.parseLong(argValue) * 1000L;
+                        break;
+                    case PROMETHUES_EXPORTER_PORT:
+                        portNumber = Integer.parseInt(argValue);
+                        break;
+                    default:
+                        System.out.println("Unrecognised argument: '" + arg + "'");
+//                        return;
+                }
+            }
+        }
+        if (portNumber == -1) {
+            System.out.println("Port to expose prometheues statistics is a compulsory argument");
+            System.out.println(
+                    "Usage: [port=<port for prometheus server COMPULSORY>]" +
+                            "\t[delay=<seconds between updates>]%n" +
+                            "\t[-Daeron.dir=<directory containing CnC file>] ");
```

Comment:
The same message is printed in two places - here and in _checkForHelp_ function. Could you either move it to a function or use Apache CLI which will print a help message for you?

Reply:
Also, please rename _Daeron.dir_ to _aeron.dir_

Reply:
Done